README.md

```diff
@@ -1,30 +1,39 @@
-<h1 align="center">OpenST - Empowering Decentralized Economies</h1>
+<h1 align="center">Branded Tokens - Tokenizing Mainstream Applications</h1>
 
-[![Gitter: JOIN CHAT](https://img.shields.io/badge/gitter-JOIN%20CHAT-brightgreen.svg)](https://gitter.im/OpenSTFoundation/SimpleToken)
+[![Discourse: JOIN DISCUSSION](https://img.shields.io/discourse/https/discuss.openst.org/topics.svg?style=flat)](https://discuss.openst.org/)
 
-OpenST blockchain infrastructure empowers new economies for mainstream businesses and emerging (D)Apps. The token and other smart contracts in this repository are intended for use with the OpenST Protocol, a framework for tokenizing businesses.
+A Branded Token allows a mainstream application to create a value-backed token designed specifically for its application's context. A Branded Token implements the required and optional [EIP-20 Standard Token interface](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md).
 
-_While this software is available as-is for anyone to use, we caution that this is in early stage and under heavy ongoing development and improvement. Please report bugs and suggested improvements._
+Holders of branded tokens must be able to redeem the value that backs those branded tokens. Branded tokens are only usable within the application context. The application can maintain a policy on accepting new holders.
 
-## OpenST Tokens
+A Utility Branded Token, which also implements the EIP-20 Standard Token interface, is the utility token representation of the Branded Token on a sidechain. This representation is orchestrated via a gateway, comprising a Gateway contract on the same chain as the Branded Token, and a CoGateway contract on the sidechain with the Utility Branded Token.
 
-Utility Branded Tokens and Value Branded Tokens form the basis of token economies built with the OpenST Protocol.
+A Gateway Composer facilitate's staking value for branded tokens and minting a utility representation of those branded tokens with a Utility Branded Token through a gateway for use within the given application.
```

Comment:
Do you think we should append below:

"GatewayComposer is a composition contract which can be used to optimise the flow of the user where the user intends to perform a single combined action."

Reply:
Yeah, that should be there. Will update.